src/screens/okr/components/key-result/key-result.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { View } from 'react-native';
+
+import { IKeyResult } from 'src/common/types';
+import { Text } from 'src/components';
+import styles from './styles';
+
+type KeyResultProps = {
+  keyResult: IKeyResult;
+};
+
+const keyResult: React.FC<KeyResultProps> = ({
+  keyResult: { name, points },
+}) => {
```

Comment:
```suggestion
const keyResult: React.FC<KeyResultProps> = ({ name, points }) => {
```
As long as we don't expect anything but keyResult itself here